msg/AutotuneAttitudeControlStatus.msg

```diff
@@ -1,8 +1,10 @@
-uint64 timestamp                # time since system start (microseconds)
+# Autotune attitude control status
 
-float32[5] coeff                # coefficients of the identified discrete-time model
-float32[5] coeff_var            # coefficients' variance of the identified discrete-time model
-float32 fitness                 # fitness of the parameter estimate
+uint64 timestamp  # [us] Time since system start
+
+float32[5] coeff      # [-] Coefficients of the identified discrete-time model
```

Comment:
FYI, 
- `[x]` means "units", and `[-]` specifically means "no units". Would hope for units for the other fields as well.
- Comment indentation is done by block of field and aligns two spaces after the longest field name.

Reply:
@mahima-yoga Thanks for the update. Looks great. Resolving!